### DISCUSSION TITLE
TypeError: Cannot read property 'access' of undefined

### DISCUSSION THREAD

**Discussion by vinaydixit83** [vercel/next.js#15347] [Help]
I am trying to learn Next js. But it is giving me some unusual error. On google also i could not find anything.

![image](https://user-images.githubusercontent.com/34070183/87990296-530f0b00-caa9-11ea-8834-bf00c866609f.png)
 I am nor sure what is the problem here.
I am following the steps available on udemy.com.
Please suggests.

**Comment by Timer** [comment]
Please upgrade your Node.js version to the latest LTS!
